packages/render-fidelity-tools/src/workflows/test-fidelity.ts

```diff
@@ -58,5 +62,31 @@ screenshotCreator.captureAndAnalyzeScreenshots(scenarioWhitelist)
     .then(() => {
       console.log(`✅ Results recorded to ${outputDirectory}`);
       server.close();
+
+      const modelViewerErrorPath =
+          join(outputDirectory, 'modelViewerFidelityErrors.json');
+      const modelViewerFidelityErrors = require(modelViewerErrorPath);
+
+      // config contains all scenarios, testConfig contains only scenarios that
+      // the test run on.
+      const testConfigPath = join(outputDirectory, 'config.json');
+      const testConfig = require(testConfigPath);
+      const scenarioCount = testConfig.scenarios.length,
+            failCount = modelViewerFidelityErrors.length,
```

Comment:
This syntax is a little weird to me; let's just put each const on its own line and skip the commas.